Dockerfile

```diff
@@ -48,13 +48,15 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
 #
 
 RUN apt-get install -y python && \
-    wget -qO- http://pyyaml.org/download/pyyaml/PyYAML-3.11.tar.gz | \
+    wget -qO- https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz#md5=f50e08ef0fe55178479d3a618efe21db | \
     tar xvz --strip-components 1 && \
```

Comment:
Why is this different than the other python dependencies. I think you should be doing a `pip install` instead.